src/mutating-policies.md

```diff
@@ -0,0 +1,129 @@
+### Mutating policies validation
```

Comment:
Nitpicking:

```suggestion
### Mutating policies
```
(or something similar) to remove ambiguity; so people don't confuse that we are validating the yaml of the policy definition.